spring-cloud-commons/src/test/java/org/springframework/cloud/client/loadbalancer/reactive/DefaultReactiveLoadBalancer.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2012-2019 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.springframework.cloud.client.loadbalancer.reactive;
+
+import java.util.Random;
+
+import org.reactivestreams.Publisher;
+import reactor.core.publisher.Mono;
+
+import org.springframework.cloud.client.DefaultServiceInstance;
+import org.springframework.cloud.client.ServiceInstance;
+
+/**
+ * A sample implementation of {@link ReactiveLoadBalancer} used for tests.
+ *
+ * @author Olga Maciaszek-Sharma
+ */
+class DefaultReactiveLoadBalancer implements ReactiveLoadBalancer<ServiceInstance> {
```

Comment:
How about `TestReactiveLoadbalancer`? `Default` makes me think it should not be in test :-)

Reply:
yes :) I think I had another `TestReactiveLoadBalancer` and I've named this one differently, but, yes, it does not make much sense, since they have different signatures anyway. Changing it.